packages/vue-dot/src/patterns/FilterModule/FilterModule.vue

```diff
@@ -120,6 +120,8 @@
 
 			if (this.filtersCount === 0) {
 				this.resetFilters();
+			} else {
+				this.emitFilterListEvent();
 			}
```

Comment:
```suggestion
			if (this.filtersCount === 0) {
				this.resetFilters();
				return;
			}

			this.emitFilterListEvent();
```
de cette manière ça suit la syntaxe utilisée plus haut dans la fonction